source/common/thread_safe_monotonic_time_stopwatch.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "nighthawk/common/stopwatch.h"
+
+#include "external/envoy/source/common/common/lock_guard.h"
+#include "external/envoy/source/common/common/thread.h"
+
+namespace Nighthawk {
+
+/**
+ *  Utility class for thread safe tracking of elapsed monotonic time.
+ */
```

Comment:
(nit) Can we include an example use of this class? Either here, or where the interface is defined.